diff start->xxx
diff elapsed time->xxx
lhs->test23.lhs.csv
rhs->test23.rhs.csv
sink->test23.sink.diff
kind->BOTH
maxDiffs->9223372036854775807
diff_column_names=[COLUMN5]

--- vhl summary ---
diff'd 8 rows in xxx, found:
!4 row diffs
@7 column diffs
-------------------
--- row diff summary ---
1 row diffs <
3 row diffs >
------------------------
--- column diff summary ---
columns having diffs->(COLUMN3, COLUMN4, COLUMN2)
COLUMN3 has 4 diffs
COLUMN4 has 2 diffs
COLUMN2 has 1 diffs
---------------------------
--- column diffs clustered ---
columnClusters having diffs->(COLUMN3, COLUMN2.COLUMN3.COLUMN4, COLUMN3.COLUMN4)
COLUMN3 has 2 diffs
COLUMN2.COLUMN3.COLUMN4 has 1 diffs
COLUMN3.COLUMN4 has 1 diffs
---------------------------
--- row diff groups ---
1 groups <
2 groups >
- <
COLUMN2=0000,COLUMN3=x has 1 diffs
- >
COLUMN2=1111,COLUMN3=x has 2 diffs
COLUMN2=0000,COLUMN3=v has 1 diffs
------------------------
--- column diff groups ---
COLUMN3=z:x has 2 diffs
COLUMN2=0000:3333 has 1 diffs
COLUMN3=u:x has 1 diffs
COLUMN3=y:x has 1 diffs
------------------------
--- column diff cluster groups ---
COLUMN2.COLUMN3.COLUMN4/COLUMN2=0000:3333,COLUMN3=z:x has 1 diffs
COLUMN3.COLUMN4/COLUMN2=4444,COLUMN3=z:x has 1 diffs
COLUMN3/COLUMN2=2222,COLUMN3=y:x has 1 diffs
COLUMN3/COLUMN2=5555,COLUMN3=u:x has 1 diffs
------------------------
!{COLUMN1=1, COLUMN2=0000, COLUMN3=x}
<
!{COLUMN1=2, COLUMN2=1111, COLUMN3=x}
>
@{COLUMN1=3, COLUMN2=2222, COLUMN3=y:x}
COLUMN3
<y
>x
@{COLUMN1=4, COLUMN2=0000:3333, COLUMN3=z:x}
COLUMN2
<0000
>3333
COLUMN3
<z
>x
COLUMN4
<bbbb
>aaaa
@{COLUMN1=5, COLUMN2=4444, COLUMN3=z:x}
COLUMN3
<z
>x
COLUMN4
<bbbb
>aaaa
@{COLUMN1=6, COLUMN2=5555, COLUMN3=u:x}
COLUMN3
<u
>x
!{COLUMN1=7, COLUMN2=0000, COLUMN3=v}
>
!{COLUMN1=8, COLUMN2=1111, COLUMN3=x}
>
